### PR TITLE
feat(group-index): Return activities in issue page

### PR DESCRIPTION
**Pull request by ahmedetefy**

### CODE REVIEW

File: src/sentry/api/helpers/group_index.py [hunk 1]
```diff
@@ -950,6 +950,20 @@ def update_groups(request, group_ids, projects, organization_id, search_fn):
...
+            if res_type in (
+                GroupResolution.Type.in_next_release,
+                GroupResolution.Type.in_release,
+            ):
+                result["activity"] = serialize(
+                    Activity.get_activities_for_group(group=group_list[0], num=100), acting_user
+                )
+    except UnboundLocalError:
+        ...
+
```
Comment (iProgramStuff) [inline src/sentry/api/helpers/group_index.py:966]: 👀

**Review by iProgramStuff — APPROVED** [review]
Tested locally and seems to work as expected. Added a couple checks in the tests to verify as well.

Wondering why we don't return the activity for all updates though? Not sure where this is being used on the frontend

**Review by wedamija — COMMENTED** [review]
I'd recommend just making this a separate endpoint - `group_activity.py`, and call `get_activities_for_group` there.

File: src/sentry/api/helpers/group_index.py (hunk 1 above)
Comment (ahmedetefy) [reply src/sentry/api/helpers/group_index.py]: fwiw `...` same as `pass`

**Comment by ahmedetefy** [comment]
Thanks @iProgramStuff for taking over.. I saw your Jira ticket on creating the end point .. will get to it

**Pull request merged**